tactics/eClause.ml

```diff
@@ -49,14 +49,7 @@ let check_bindings bl =
               str " occurs more than once in binding list.")
     | [] -> ()
 
-let explain_no_such_bound_variable mvl {CAst.v=id;loc} =
-  let open Pp in
-  let expl = match mvl with
-  | [] -> str "(no bound variables at all in the expression)."
-  | [id] -> str "(possible name is: " ++ Id.print id ++ str ")."
-  | _ -> str "(possible names are: " ++ pr_enum Id.print mvl ++ str ")."
-  in
-  user_err ?loc (str "No such bound variable " ++ Id.print id ++ spc () ++ expl)
+let explain_no_such_bound_variable = Clenv.explain_no_such_bound_variable
```

Comment:
Not relying on the Clenv module was an explicit goal of code duplication here.

Reply:
What to do? Moving the shared code elsewhere?

Reply:
I restored the two copies (assuming that `clenv.ml` is supposed to disappear relatively soon).